cmd/cadvisor.go

```diff
@@ -77,6 +77,8 @@ var rawCgroupPrefixWhiteList = flag.String("raw_cgroup_prefix_whitelist", "", "A
 
 var perfEvents = flag.String("perf_events_config", "", "Path to a JSON file containing configuration of perf events to measure. Empty value disabled perf events measuring.")
 
+var resctrlInterval = flag.Duration("resctrl_interval", 0, "Resctrl mon groups updating interval. Zero value disables updating mon groups.")
```

Comment:
I mean, watching mon groups and making sure that have correct pids. 
Thanks to that flag we can monitor containers that changed their control group during the cAdvisor run. 

Setting `--resctrl_interval` to 0, disables updating mon groups. 
This is good behavior for users that want to use only monitoring features of Resctrl.

We cannot depend on `--housekeeping_interval` because cAdvisor will be updating containers instantly:
```
...
I0315 12:54:47.268474   22054 container.go:593] [/some_cgroup] Housekeeping took 331.572µs
I0315 12:54:47.268948   22054 container.go:593] [/some_cgroup] Housekeeping took 429.79µs
I0315 12:54:47.269348   22054 container.go:593] [/some_cgroup] Housekeeping took 356.188µs
I0315 12:54:47.269363   22054 container.go:593] [/machine.slice/libpod-conmon-e361822894402b35f7230ccd133725e8d52e97c8eddd3ba77cd0dc3fc9ee46ad.scope] Housekeeping took 1.506114ms
I0315 12:54:47.269385   22054 container.go:593] [/system.slice/thermos.service] Housekeeping took 1.48471ms
I0315 12:54:47.269919   22054 container.go:593] [/some_cgroup] Housekeeping took 528.066µs
I0315 12:54:47.270333   22054 container.go:593] [/some_cgroup] Housekeeping took 371.664µs
I0315 12:54:47.270611   22054 container.go:593] [/some_cgroup] Housekeeping took 220.11µs
I0315 12:54:47.270855   22054 container.go:593] [/some_cgroup] Housekeeping took 188.26µs
I0315 12:54:47.271074   22054 container.go:593] [/machine.slice/libpod-conmon-df33bd5e05ec42afd4c39cc0a1417c227553de2bd7fc510de00fb7341d59d263.scope] Housekeeping took 1.682142ms
I0315 12:54:47.271163   22054 container.go:593] [/some_cgroup] Housekeeping took 250.002µs
I0315 12:54:47.271407   22054 container.go:593] [/some_cgroup] Housekeeping took 185.349µs
...
```

Reply:
If something will change the control group, the cAdvisor will continue measuring in the new group.
cAdvisor manipulates only mon groups. Now it's aware of existing groups, even own prepared have "cadvisor" prefix,
e.g. `cadvisor-system.slice-docker-0f07bb8ed38c0f40c9424875f2cf417d19b27ca93b631640a2039b3478ee5858.scope`
So will manipulate only the groups with that prefix.